modules/tinymce/src/core/main/ts/api/OptionTypes.ts

```diff
@@ -69,6 +69,8 @@ interface BaseEditorOptions {
   cache_suffix?: string;
   color_cols?: number;
   color_map?: string[];
+  color_foreground_default?: string;
+  color_background_default?: string;
```

Comment:
Should this instead be called `color_default_foreground` and `color_default_background`? As this is inconsistently named to everything else... for example `color_map_background`, `color_cols_foreground` from https://github.com/tinymce/tinymce/pull/8157.

Either way you and @HAFRMO need to standardise the naming here as atm it's inconsistent.

- [ ] Ensure the option names are consistent.

Reply:
I prefer `color_default_*` and `color_map_*`